doc/contrib/docs.rst

```diff
@@ -8,23 +8,73 @@ Documentation and Examples
   :backlinks: none
   :local:
 
-*********
-Documents
-*********
+*************
+Documentation
+*************
 * Python and C documentation is built using `Sphinx <http://www.sphinx-doc.org/en/master/>`_.
 * Each document is written in `reStructuredText <http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_.
-* You can build document locally to see the effect, by running
+* The documentation is the ``doc/`` directory.
+* You can build it locally using ``make html`` command.
 
-.. code-block:: bash
+   .. code-block:: bash
 
-  make html
+     make html
 
-inside the ``doc/`` directory.  The online document is hosted by `Read the Docs <https://readthedocs.org/>`__ where the imported project is managed by `Hyunsu Cho <https://github.com/hcho3>`__ and `Jiaming Yuan <https://github.com/trivialfis>`__.
+The online document is hosted by `Read the Docs <https://readthedocs.org/>`__ where the imported project is managed by `Hyunsu Cho <https://github.com/hcho3>`__ and `Jiaming Yuan <https://github.com/trivialfis>`__.
+
+===============================
+Build Docs on macOS using Conda
```

Comment:
Since you use conda for illustration, it should work across various platforms. We just need to replace the `brew` step with `graphviz` from conda forge.

Aside from this, let's make it clear that this is building the HTML document.